src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-how-nrql-works.mdx

```diff
@@ -103,11 +103,11 @@ NRQL's basic rules are as follows:
 
     <tr>
       <td>
-        Attribute names with spaces
+        Non-standard custom event and attribute names
       </td>
 
       <td>
-        Use backticks ``` `` ``` to quote a [custom attribute](/docs/insights/new-relic-insights/decorating-events/insights-custom-attributes) name that has a space in it. For example:
+        Events and attributes that we report by default have names that contain alphanumeric characters, colons (`:`), periods (`.`), and underscores (`_`). Default-reported names also will start with an alphabetical letter, and won't have spaces. For custom names that don't meet these rules, you must use backticks ``` `` ``` around them. For example:
```

Comment:
How about, "Events and attributes that we report by default have names that contain alphanumeric characters, colons (`:`), periods (`.`), and underscores (`_`). Default-reported names will start with a letter, and won't include nonstandard characters. Custom names that don't follow these guidelines must be enclosed with backticks ``` `` ``` in NRQL queries."